.github/workflows/matrix-and-codecov-on-merge-to-main.yml

```diff
@@ -42,7 +42,17 @@ jobs:
           conda config --set always_yes yes
           --set changeps1 no
 
-      - name: Install diffpy.labpdfproc and requirements
+      - name: Install diffpy.labpdfproc and requirements for Linux
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libgtk-3-dev
```

Comment:
This is needed for`linux` runner in this CI to build `wxpython` from the sdist file. This is not needed for `Windows` and `macOS`.